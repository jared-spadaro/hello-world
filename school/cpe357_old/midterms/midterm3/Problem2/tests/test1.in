0x000000AA 3 4 0x0000000E
