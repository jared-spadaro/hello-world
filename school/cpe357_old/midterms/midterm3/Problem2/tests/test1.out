0x000000F2
